php/libraries/BreadcrumbTrail.class.inc

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This file represent a BreadcrumbTrail object
```

Comment:
Could use a little more description of why you would use a BreadcrumbTrail object and not just a Breadcrumb[] array.